idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -298,6 +298,21 @@ service WorkflowService {
       4: shared.ServiceBusyError serviceBusyError,
     )
 
+  /**
+  * SignalWithStartWorkflowExecution is used to ensure sending a signal event to a workflow execution.
+  * If workflow is running, this results in WorkflowExecutionSignaled event recorded in the history
+  * and a decision task being created for the execution.
+  * If workflow is not running or not found, this results in WorkflowExecutionStarted and WorkflowExecutionSignaled
+  * event recorded in history, and a decision task being created for the execution
```

Comment:
event recorded in history, and a decision task being created for the execution -> events recorded in the history, and a decision task being created for the execution.